modules/puzzles/module.py

```diff
@@ -52,3 +53,40 @@ def buildings_with(self, symbols):
         results = util.buildings_containing_symbols(symbols)
         self.respond('Er zijn %i gebouwen die de symbolen \'%s\' bevatten' % (len(results), symbols))
         self.respond(', '.join([str(x) for x in results]))
+
+
+class Words(TelegramModule):
+    @command
+    def text_to_morse(self, *words):
+        """
+        Zet een aantal woorden om naar morse
+        """
+        try:
+            self.respond('\n'.join([' '.join(CODE[i.upper()] for i in word)
+                                   for word in words]))
+        except KeyError as e:
+            self.respond('Character %s cannot be parsed to morse, only characters and digits can'
+                         % str(e))
+
+    @command
+    def morse_to_text(self, *morse):
+        """
+        Zet een aantal morse letters (onderbroken door spatie) en woorden (onderbroken door ;)
```

Comment:
Now using `;` as word breaker, am not sure if that is the best. An enter cannot be used since the method just sees enters as spaces